api/google-search/pom.xml

```diff
@@ -67,6 +68,45 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
```

Comment:
@jpukg why `jacoco` chosen as plugin instead of `dependency`.

Reply:
My thoughts: Code coverage is not essential for running/executing the application so it's not dependency. It's a plugin because it supports Maven to generated coverage reports by running through the code internally.

Reply:
@ramsunvtech - @shangan23 explained well.